dashboard/app/controllers/foorm/forms_controller.rb

```diff
@@ -0,0 +1,82 @@
+module Foorm
+  class FormsController < ApplicationController
+    before_action :require_levelbuilder_mode_or_test_env
+    before_action :authenticate_user!
+    load_and_authorize_resource
+
+    # PUT foorm/form/:id/edit_questions
+    def edit_questions
```

Comment:
why not use the standard `update` method here?

Reply:
I don't want to ever allow updating of name or version, since that will write to a new file and on seed the old file will come back into the table, so I wanted to make it clear this is just for updating questions.